docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -6,18 +6,16 @@ title: Persistent Volume Claim Protection
 ---
 
 {% capture overview %}
-{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+{% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
 
-As of Kubernetes 1.9, persistent volume claims (PVCs) that are in active use by a pod can be protected from pre-mature removal.
+As of Kubernetes 1.10, persistent volume claims (PVCs) that are in active use by a pod can be protected from pre-mature removal.
```

Comment:
Do we need to mention the version here since there is a tag for it?

It's a little confusing to say as of 1.10 because it was technically available since 1.9